Formula/embree.rb

```diff
@@ -0,0 +1,45 @@
+class Embree < Formula
+  desc "High-performance ray tracing kernels"
+  homepage "https://embree.github.io/"
+  url "https://github.com/embree/embree/archive/v3.5.0.tar.gz"
+  sha256 "4635439c277d0f688f1a21ad13ad22fdadbcbca04680cce346c88179b9522741"
+  head "https://github.com/embree/embree.git"
+
+  depends_on "cmake" => :build
+  depends_on "ispc" => :build
+  depends_on "tbb"
+
+  def install
+    args = std_cmake_args + %w[
+      -DBUILD_TESTING=OFF
+      -DEMBREE_IGNORE_CMAKE_CXX_FLAGS=OFF
+      -DEMBREE_ISPC_SUPPORT=ON
+      -DEMBREE_MAX_ISA=SSE4.2
+      -DEMBREE_TUTORIALS=OFF
+    ]
+
+    mkdir "build" do
+      system "cmake", *args, ".."
+      system "make"
+      system "make", "install"
+    end
+
+    # Remove models directory and the resultant empty bin directory since
+    # tutorials are not enabled.
+    rm_rf bin
```

Comment:
The models directory is not removed. The empty bin directory will probably be removed by Homebrew automatically.

Reply:
Neither of your statements are true, actually.